curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fizzbuzz.english.md

```diff
@@ -0,0 +1,92 @@
+---
+title: FizzBuzz
+id: 5e9ddb06ec35240f39657419
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+
+Write a program that outputs the integers from 1 to 100 (inclusive). But: <br>
+<ul>
+    <li> for multiples of 3, output <b>Fizz</b> instead of the number </li>
+    <li> for multiples of 5, output <b>Buzz</b> instead of the number </li>
+    <li> For multiples of 3 and 5, output <b>FizzBuzz</b> instead of the number </li>
+</ul>
+
+</section>
+
+## Instructions
+<section id='instructions'>
+
+Your program should return an array containing the results based on the rules above.
+
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: <code>fizzBuzz</code> should be a function.
+    testString: assert(typeof fizzBuzz=='function');
+  - text: <code>fizzBuzz()</code> should return an Array.
+    testString: assert(Array.isArray(fizzBuzz())==true);
+  - text: Number only divisible by 3 should return <code>"Fizz"</code>.
+    testString: assert.equal(fizzBuzz()[2], "Fizz");
+  - text: Numbers only divisible by 5 should return <code>"Buzz"</code>.
+    testString: assert.equal(fizzBuzz()[99], "Buzz");
+  - text: Numbers divisible by 3 and 5 should return <code>"FizzBuzz"</code>.
+    testString: assert.equal(fizzBuzz()[89], "FizzBuzz");
+  - text: Numbers not divisible by 3 or 5 should return the number itself.
+    testString: assert.equal(fizzBuzz()[12], 13);
+
```

Comment:
I don't like how we're doing this though. A user can just hard code these values. I have an alternative approach, I'll put it below